internal/proposals/logs/dynamic-backend-configuration/custom-operator/dynamic-fluentbit-config.md

```diff
@@ -0,0 +1,211 @@
+
+## Criteria
+- Customers can place fluent-bit config snippets as k8s resource in any namespace.
+- The customers' configuration must not be reset during reconciliation, even though the central configuration might be overwritten at any time.
+- A new configuration is picked up dynamically without any user interaction (for example, no need that users restart pods).
+- Have basic validation to give early feedback on errors, which supports debugging a startup problem.
+- The proposal should be extensible to solve the same issue for OpenTelemetry collector.
+- There's a way to provide the auth details for a backend config in a secure way.
+
+## Proposal
+
+### Architecture
+
+To build the operator, we use an SDK for a quick and easy start-up, among other reasons. We decided to use [`kubebuilder`](https://github.com/kubernetes-sigs/kubebuilder), an upstream project of `Kubernetes`. `kubebuilder` provides the required features for our operator with the smallest overhead. In addition, this `kubebuilder` is also used in other parts of Kyma, ensuring consistent design.
+
+An alternative for `kubebuilder` is the [`Operator-SDK`](https://github.com/operator-framework/operator-sdk) from `Red Hat`. `Operator-SDK` has similar features to the `kubebuilder`, with some additions like the integration of [`operatorhub`](https://operatorhub.io/). We decided against it because such features aren't needed for our purpose, and because it isn't an upstream project of Kubernetes.
+
+To have a simple API, one `CRD` for Fluent Bit configuration is created. This CRD has a field which holds the status of the CR, called `Status`, as well as a struct of the type `LoggingConfigurationSpec`, called `Spec`, holding a list of configuration sections, called `Sections`. Each `Section` determines the type of the configuration (`FILTER` or `OUTPUT`) and hold a list for configuration entries, called `Entries`. Each `Entry` has tree values: The `Name`, determining the key for the value, the `Value`, and the `SecretValue`. Eather the `Value` or the `SecretValue` should be used by the users for one `Name`. If no `Secret` needs to be stated for a `Name`, then `Value` should be choosen. If a `Secret` needs to be state, i.e. for an API key, then a name, the namespace of the key, and the key itself needs to be stated.
+
+Using this structure enables the possibility to support the full Fluent Bit syntax without having the need to maintain new features of various plugins. Furthermore, this gives the users the ability to have a good overview of their sequence of applied filters and outputs. Using the Kyma documentation, we could also lead the users to think more in a way of pipelines, in such that they create one CR for each Fluent Bit pipeline.
+
+<details>
+<summary><b>Pipeline Overview</b> for User - Click to expand</summary>
+
+![Thank you](images/fluentbit_CR_overview.svg)
+</details> 
+
+We're evaluating the following constraints of this custom operator: 
```

Comment:
I may have misunderstood this in my previous review. As I thought we're talking about kubebuilder, I believed the constraints must be mitigated. Now I see those are constraints that we want to impose. I believe this might be more correct?

```suggestion
We're proposing the following constraints for the custom operator: 
```